model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli.py

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2018 Olli Huotari
```

Comment:
This file was created by us. Only the source model file comes from the other author.

So this one can contain:
```python
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```